### PR TITLE
Auth with Privy, navbar and sidebar

### LINKED ISSUE #5: Research email wallet creation and integrate it
- Should be compatible with wagmi

### LINKED ISSUE #6: Setup wagmi
- Injected provider (default)
- WalletConnect

### LINKED ISSUE #8: Basic frontend - Navbar and sidebar
- Navbar with logo and wallet connection button
- Sidebar with items (my drive, contacts, trash, shared with me)

### LINKED ISSUE #24: Auth - Dedicated auth wrapper
- Avoid access to the drive page if not connected, a layout should display the login page
- Keep the typing inside the drive to always have an account defined

### PR DESCRIPTION
**Pull request by RezaRahemtola**
Fixes #6 
Fixes #5 
Fixes #8 
Fixes #24

### CODE REVIEW

File: front/.yarnrc.yml [hunk 1]
```diff
@@ -1 +1,3 @@
+nodeLinker: node-modules
+
```
Comment (EdenComp) [inline front/.yarnrc.yml:2]: Aren't we supposed to use the yarn cache?

**Review by EdenComp — DISMISSED** [review]

File: front/.yarnrc.yml (hunk 1 above)
Comment (RezaRahemtola) [reply front/.yarnrc.yml]: fixed

Comment (RezaRahemtola) [reply front/.yarnrc.yml:2]: Nope, we need to keep it that way or `yarn` won't install all dependencies and build fails.
Try to build it locally without this property, you'll need to explicitly install some peer/sub dependencies.

And this still leverages the cache on installation.

Comment (RezaRahemtola) [reply front/.yarnrc.yml:2]: cf https://yarnpkg.com/features/linkers

**Review by chuipagro — APPROVED** [review]

**Pull request merged**